Server/JokesOnYou.Web.Api/Controllers/JokesController.cs

```diff
@@ -15,5 +21,24 @@ public JokesController(IJokesService jokesService)
         {
             _jokesService = jokesService;
         }
+
+        [Authorize(Roles = "registered,admin")]
+        [HttpPost]
+        public async Task<ActionResult<JokeReplyDto>> CreateJokeAsync(JokeCreateDto jokeCreateDto)
+        {
+
+            var userId = ClaimsPrincipalExtension.GetUserId(User);
```

Comment:
is this a good way of getting the ID? We could combine it with the caller (website) actually supplying the User Id as well?
checking if both are the same ?